crates/extensions/tedge_mqtt_ext/src/test_helpers.rs

```diff
@@ -47,9 +47,7 @@ pub fn assert_message_contains_str(message: &Message, expected: (&str, &str)) {
     let payload = message.payload_str().expect("non UTF-8 payload");
     assert!(
         payload.contains(expected_payload),
-        "Payload assertion failed.\n Actual: {} \n Expected: {}",
-        payload,
-        expected_payload
+        "Payload assertion failed.\n Actual: {payload:?} \n Expected: {expected_payload:?}",
```

Comment:
I added debug printing here because the assertion message was very unclear when it failed due to a newline in the expected payload.